swap.go

```diff
@@ -11,7 +11,18 @@ import (
 )
 
 type swapKit struct {
+	// htlc stores the confirmed htlc when looping in.
+	// When looping out, it simply stores the p2wsh htlc.
 	htlc *swap.Htlc
+
```

Comment:
I think a better option may be removing `swapKit` altogether. It wasn't really that useful as we only used it to send updates and create the htlc. Rather than have `swapKit` as a base for `loopInSwap` and `loopOutSwap` we could rely on a common interface and put `SwapInfo()` and `SendUpdate()` there instead.

Reply:
Perhaps we could keep `swapKit` but move functions and htlc(s) to loopin/loopout.

Reply:
So decided to move `htlc` as well as `sendUpdate` out of `swapKit`. This way extending loop-in is more natural, while loop-out can stay as is.